content/updates/2018-02.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 2018.2.0
+date: 2018-03-07
+hidden: true
+image: TODO
+---
+
+# 2018.02 Changelog
```

Comment:
Please remove this title, it's handled by the page template itself

Reply:
fixed in fdde4951f0d127a913b0e1fb21a347f2332551f4